src/machine/usb/desc_mimxrt1062.go

```diff
@@ -16,7 +17,7 @@ const descCDCACMCount = 1
 
 // descHIDCount defines the number of USB cores that may be configured as a
 // composite (keyboard + mouse + joystick) human interface device (HID).
-const descHIDCount = 0
+const descHIDCount = 1
```

Comment:
The only solutions I can think of to this:

1. `tinygo` does an in-place file edit:
	- Would need a robust technique to find and edit this and other consts.
	- Would need to decide if/how to restore the edits afterwards. 
		- Restoring the edits will confuse IDEs/debuggers that are comparing the source to the compiled executable.
		- <ins>NOT</ins> restoring the edits means compiling source code is a _destructive_ operation on that source code!
	- Kinda _**hacky**_ IMO
2. Move these respective constants to their own standalone source files with appropriate build tag constraints:
	- Let `tinygo` adjust active build tags automatically if command-line flag given.
		- For example: `GOFLAGS="-tags=...,usb.hid,...` if command-line flag `-usb=hid` was given.

Of course, 2. would be my preference and also seems the easiest to implement.

Reply:
Note that if 2. would be preferable, it will be necessary to add a separate command-line flag to `tinygo` — piggybacking off of the `-serial` flag is insufficient.

Something like the following, along with logic to verify the flags given are compatible (e.g., `-serial=usb -usb=hid` contradicts itself)

```
tinygo -serial={none,{uart,usb}[:PORT]} -usb={cdc,hid}[:PORT]
```